### DISCUSSION TITLE
Quarkus 3.15 - new LTS version

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#43528] [Quarkus Blog/Website]
# Quarkus 3.15 - new LTS version

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-15-1-released/

**Comment by adampoplawski** [comment]
https://mvnrepository.com/artifact/io.quarkus/quarkus-bom

I do not see 3.15.1. Will be it released soon or I misunderstood announcement?

**Reply by adampoplawski** [reply]
https://github.com/quarkusio/quarkus-platform/tags platform is tagged already

**Reply by gsmet** [reply]
mvnrepository.com lags quite a bit as they are probably syncing regularly with a cronjob.

If you have a look at https://central.sonatype.com/artifact/io.quarkus/quarkus-bom, it's already there. And even this reference website can lag sometimes.
The source of truth is: https://repo1.maven.org/maven2/io/quarkus/quarkus-bom/3.15.1/ .

**Comment by marcelstoer** [comment]
Any idea by when we can expect RHBQ 3.15? I see that you only just released a [3.8.6](https://maven.repository.redhat.com/ga/com/redhat/quarkus/platform/quarkus-bom/) 3d ago.

**Reply by gsmet** [reply]
That's a question to ask through your usual Red Hat channels :).

**Reply by marcelstoer** [reply]
Thanks, I will. It's just those channels are not as responsive as you guys.

**Reply by tqvarnst** [reply]
If you are looking for public information, please check https://access.redhat.com/articles/6644111#red-hat-build-of-quarkus-3.

**Reply by marcelstoer** [reply]
Thank you! This is what I was looking for

> The Red Hat build of Quarkus 3.15 is planned to be released in Q4 2024.

**Comment by alex-kovalenko1982** [comment]
Hi, we are heavily waiting for hibernate 6.6.1 with a fix for duplicated foreign keys. 
As I see hibernate version is going to be updated in Quarkus 3.15.2.
Any estimation when 3.15.2 might be released?
Thanks!

**Reply by eduleon** [reply]
Waiting on 3.15.2 too. We also have a Hibernate issue related with embedded keys.

**Reply by alex-kovalenko1982** [reply]
@eduleon it seems that quarkus 3.16 is available now and has hibernate updated from 6.6.0 to 6.6.1. testing it now, unfortunately it brings some other hibernate-related issues. could you elaborate what is exactly the issue with embedded keys?

**Reply by eduleon** [reply]
the app doesn't work if you use the @EmbeddedId. you got a huge exception

**Comment by tqvarnst** [comment]
Hi @alex-kovalenko1982, 
I'm assuming that you are asking about the community version. In that case, I believe that Quarkus 3.15.2 should be available at the beginning of November, but @gsmet should confirm the community plans first.

**Reply by tmonney** [reply]
We are also eagerly waiting on 3.15.2 too, 3.8.x has unfixed vulnerabilities and we can't use 3.15.1 because of https://github.com/quarkusio/quarkus/issues/43651.

**Reply by gsmet** [reply]
Quarkus 3.15.2 will be released at the beginning of next week.

**Reply by tmonney** [reply]
Great news, thanks for the update.
